src/lib/pickles/wrap_hack.ml

```diff
@@ -89,12 +91,13 @@ module Checked = struct
            Dummy.Ipa.Wrap.challenges_computed )
       chalss
 
+  (** pads `commitments` up to length 2 *)
   let pad_commitments (commitments : _ Vector.t) =
-    pad_vector
-      ~dummy:
-        (Tuple_lib.Double.map ~f:Impls.Step.Field.constant
-           (Lazy.force Dummy.Ipa.Wrap.sg) )
-      commitments
+    (* use dummy sg to pad the commitments vector to a length 2 *)
```

Comment:
nit: should say `use dummy sg to pad the commitments vector to length 2` (no "a")